package.json

```diff
@@ -51,7 +51,7 @@
     "eslint-plugin-ember": "^5.0.0",
     "ethers": "3.0.15",
     "kosmos-schemas": "^1.1.2",
-    "kredits-contracts": "github:67P/truffle-kredits",
+    "kredits-contracts": "github:67P/kredits-contracts",
```

Comment:
We can actually switch to the npm package now (version 3.0.1): https://www.npmjs.com/package/kredits-contracts